daemon/main.go

```diff
@@ -138,6 +138,11 @@ func init() {
 						Name:        "keep-config",
 						Usage:       "When restoring state, keeps containers' configuration in place",
 					},
+					cli.BoolFlag{
+						Destination: &config.KVStoreIPv4Registration,
```

Comment:
Yes and that's sort of what's doing right now, the problem is, how can we know the that the node that is registering is not the same node that registered before? For example: something crashes and cilium has to be restarted again.